Jenkinsfile

```diff
@@ -190,6 +190,168 @@ pipeline {
             }
         }
 
+        stage('Build Runtime Docker for CUDA 8') {
+            agent {
+                label "nvidia-docker && (mr-dl11||mr-dl16||mr-dl10)"
+            }
+
+            steps {
+                dumpInfo 'Linux Build Info'
```

Comment:
The label does not match stage label  ;-)

Reply:
Whoops, can fix that.